megamek/src/megamek/server/ScenarioLoader.java

```diff
@@ -409,6 +446,69 @@ public IGame createGame() throws Exception {
         return g;
     }
 
+    private void parsePlanetaryConditions(Game g, StringMultiMap p) {
+        if (p.containsKey(PARAM_PLANETCOND_TEMP)) {
```

Comment:
Can we separate these if statements so that there is a whitespace between the close `}` and the next if statement?

Reply:
Also, if we are just going to use a setter immediately after can we just use the same line and combine them. Not much point to separate them out.

Reply:
a) You mean an empty line after }? parsePlanetaryConditions is so uninteresting that I wanted it to be short. No complexity here. 
b) What? You mean like g.getPlanetaryConditions().setTemperature(Integer.parseInt(p.getString(PARAM_PLANETCOND_TEMP)));?
I can do that but it makes lines very long and Id rather have them like this than with a break in the middle.

Reply:
Arlith used to run a code formatter that would cut the lines very short iirc and (I think) sometimes result in awful line breaks. I'd rather break manually. But if no one does this, I can sure use long code lines.

Reply:
Exactly like this. The extra space is an accessibility bit that assists in separating out the statements to a reader. Otherwise, it can come off as being a single giant chained if statement even though it is not the case